versioned_docs/version-8.7/apis-tools/camunda-api-rest/camunda-api-rest-overview.md

```diff
@@ -10,10 +10,6 @@ The Camunda 8 REST API is a REST API designed to interact with a Camunda 8 clust
 Ensure you [authenticate](./camunda-api-rest-authentication.md) before accessing the Camunda 8 REST API.
 :::
 
-:::note
-The Camunda 8 REST API only supports user tasks managed by Camunda (formerly known as [Zeebe user tasks](/apis-tools/migration-manuals/migrate-to-camunda-user-tasks.md), which may still appear as such in your XML content).
-:::
-
```

Comment:
I don't actually know if this should be removed. It exists in the 8.8 overview, but not the 8.6 overview.

Reply:
This was introduced with https://github.com/camunda/camunda-docs/pull/4723. In that PR, we only added the note to the "next" docs but not the 8.6 docs. @christinaausley, was that on purpose? In 8.6, the user task migration guide is still called "Zeebe user task". With 8.7, we now switch to the "Camunda user task" wording. Is that on purpose for 8.7 already or should this only come with 8.8, also regarding, for example, Camunda modeler look and feel for 8.7? @aleksander-dytko, can you clarify this from a PM perspective as well, please?

Reply:
Plus, the user task migration guide for 8.7 is "too bold" and basically what comes with 8.8, I believe. It might need to go back to the 8.6 version.

Reply:
@tmetzke thanks for highlighting that. For the `Camunda User Task` wording - we can switch over to using that as soon as possible to accustom customers to this wording and use this task type. I see in Modeler, that the name for `Zeebe User Task` has changed already for all versions. @barmac could you confirm?

Reply:
We are using "Camunda User Task" across Modelers and related docs. Confirmed.

Reply:
I'm sorry all, but I'm not clear what to do with this feedback. Can someone summarize the expected actions on my part?

Reply:
Parallel threads are happening here. Let's match the content in the 8.6 overview page, which does not include this note. This PR is correct as-is for this part of the conversation.

We've removed the user task migration guide for 8.7 in a separate PR - https://github.com/camunda/camunda-docs/pull/5155.

Resolving this thread.